siriuspy/siriuspy/clientconfigdb/types/as_rf.py

```diff
@@ -452,50 +452,87 @@ def get_dict():
+    # ['RA-ToSIA02:AlarmConfig:GeneralPowerLimHigh', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:GeneralPowerLimLow', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:GeneralPowerLimLoLo', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:InnerPowerLimHiHi', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:InnerPowerLimHigh', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:InnerPowerLimLow', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:InnerPowerLimLoLo', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:CurrentLimHiHi', 0, 0.0],  # Alarms lims currents SSA tower 2
+    # ['RA-ToSIA02:AlarmConfig:CurrentLimHigh', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:CurrentLimLow', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:CurrentLimLoLo', 0, 0.0],
+    # ['RA-ToSIA03:OffsetConfig:UpperIncidentPower', 0, 0.0],  # Offsets SSA Twr3
+    # ['RA-ToSIA03:OffsetConfig:UpperReflectedPower', 0, 0.0],
+    # ['RA-ToSIA03:OffsetConfig:LowerIncidentPower', 0, 0.0],
+    # ['RA-ToSIA03:OffsetConfig:LowerReflectedPower', 0, 0.0],
+    # ['RA-ToSIA03:OffsetConfig:InputIncidentPower', 0, 0.0],
+    # ['RA-ToSIA03:OffsetConfig:InputReflectedPower', 0, 0.0],
+    # ['RA-ToSIA03:OffsetConfig:OutputIncidentPower', 0, 0.0],
+    # ['RA-ToSIA03:OffsetConfig:OutputReflectedPower', 0, 0.0],
+    # ['RA-ToSIA03:AlarmConfig:InnerPowerLimHiHi', 0, 0.0], # Alarms lims pwr SSA tower 3
+    # ['RA-ToSIA03:AlarmConfig:InnerPowerLimHigh', 0, 0.0],
+    # ['RA-ToSIA03:AlarmConfig:InnerPowerLimLow', 0, 0.0],
+    # ['RA-ToSIA03:AlarmConfig:InnerPowerLimLoLo', 0, 0.0],
+    # ['RA-ToSIA03:AlarmConfig:CurrentLimHiHi', 0, 0.0],  # Alarms lims currents SSA tower 3
+    # ['RA-ToSIA03:AlarmConfig:CurrentLimHigh', 0, 0.0],
+    # ['RA-ToSIA03:AlarmConfig:CurrentLimLow', 0, 0.0],
+    # ['RA-ToSIA03:AlarmConfig:CurrentLimLoLo', 0, 0.0],
+    # ['RA-ToSIA04:OffsetConfig:UpperIncidentPower', 0, 0.0],  # Offsets SSA Twr4
+    # ['RA-ToSIA04:OffsetConfig:UpperReflectedPower', 0, 0.0],
+    # ['RA-ToSIA04:OffsetConfig:LowerIncidentPower', 0, 0.0],
+    # ['RA-ToSIA04:OffsetConfig:LowerReflectedPower', 0, 0.0],
+    # ['RA-ToSIA04:OffsetConfig:InputIncidentPower', 0, 0.0],
+    # ['RA-ToSIA04:OffsetConfig:InputReflectedPower', 0, 0.0],
+    # ['RA-ToSIA04:OffsetConfig:OutputIncidentPower', 0, 0.0],
+    # ['RA-ToSIA04:OffsetConfig:OutputReflectedPower', 0, 0.0],
+    # ['RA-ToSIA04:AlarmConfig:InnerPowerLimHiHi', 0, 0.0], # Alarms lims pwr SSA tower 4
+    # ['RA-ToSIA04:AlarmConfig:InnerPowerLimHigh', 0, 0.0],
+    # ['RA-ToSIA04:AlarmConfig:InnerPowerLimLow', 0, 0.0],
+    # ['RA-ToSIA04:AlarmConfig:InnerPowerLimLoLo', 0, 0.0],
+    # ['RA-ToSIA04:AlarmConfig:CurrentLimHiHi', 0, 0.0],  # Alarms lims currents SSA tower 4
+    # ['RA-ToSIA04:AlarmConfig:CurrentLimHigh', 0, 0.0],
+    # ['RA-ToSIA04:AlarmConfig:CurrentLimLow', 0, 0.0],
+    # ['RA-ToSIA04:AlarmConfig:CurrentLimLoLo', 0, 0.0],
+    ['RA-ToSIA03:RF-SSAmpTower:Hw2PwrFwdInCoeff-Cte', _np.array([0, 0, 0, 0, 0]), 0.0], #SSA3 In Pwr Fwd Cal Coeff
```

Comment:
This line is larger than 80 columns. Reduce its length.
One way of doing this is by defining the value in a variable, and putting the comment in the line above the one it reffers to.

Reply:
I think you don't need a 2D array. You could do: `az = _np.zeros(5)`.

Besides, I don't like these comments at the end of the line with a line break. 
Could you try to put the comment in the line directly above the one it refers to?

Reply:
> I think you don't need a 2D array. You could do: `az = _np.zeros(5).`

I see. I will change the array declaration.

> Besides, I don't like these comments at the end of the line with a line break.
Could you try to put the comment in the line directly above the one it refers to?

I don't like either. I thought it was some kind of recommendation... Sure, I will change the comments as well!